### DISCUSSION TITLE
望可以添加一个JSON字段值提取工具【指定字段】

### DISCUSSION THREAD

**Discussion by L-ShiYue** [he3-app/he3-feedback#329] [Feature Requests]
### Description of new tools or features

版本：最新
期望：添加一个JSON字段值提取工具【用户自定义字段】，比如我要抽取Json中所有字段KEY为“detId”的值

### System you are using

macOS silicon

**Comment by gljack** [comment]
看你这描述貌似要的是JSON Path功能，我用的是V2.0.15，这个功能已经有了

**Reply by L-ShiYue** [reply]
多谢，找到了
